samples/bynder-dialog/src/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useReducer } from 'react';
+import ReactDOM from 'react-dom';
+import { Button } from '@contentful/forma-36-react-components';
+import { init, locations } from 'contentful-ui-extensions-sdk';
+import '@contentful/forma-36-react-components/dist/styles.css';
+import './index.css';
+import extension from '../extension.json';
+import BynderDialog from './BynderDialog';
+import Thumbnail from './Thumbnail';
+
+function reducer(state, action) {
+  switch (action.type) {
+    case 'add':
+      return [...state, action.payload];
+    case 'add-all':
+      return [...state, ...action.payload];
+    case 'remove':
+      return state.filter(item => item.id !== action.payload);
+    default:
+      return state;
+  }
+}
+
+function Field({ sdk }) {
+  const [assets, dispatch] = useReducer(reducer, sdk.field.getValue() || []);
+
+  useEffect(() => {
+    sdk.window.startAutoResizer();
+    return () => {
+      sdk.window.stopAutoResizer();
+    };
+  }, []);
+
+  useEffect(() => {
+    sdk.field.setValue(assets);
+  }, [assets]);
+
+  const onButtonClick = async () => {
+    const assets = await sdk.dialogs
```

Comment:
Can we somehow get rid of the scrollbar for the dialog? Seems to be caused by improper bynder div sizes.